demo/snapshot/modal-test.html

```diff
@@ -61,6 +65,14 @@
                 setTimeout(() => {
                     paypal.Message.render(config, '.banner-4');
                 }, 1000);
+            } else if (config.account === 'DEV000000FRPL') {
```

Comment:
Can we wrap this whole `if-else` block in a single `setTimeout` and only call `render` within each if?